lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -2460,7 +2538,8 @@ namespace Js
     }
 
     template <typename T>
-    void DictionaryTypeHandlerBase<T>::InvalidateFixedField(DynamicObject* instance, PropertyId propertyId, DictionaryPropertyDescriptor<T>* descriptor)
+    template <typename TPropertyKey>
```

Comment:
T is for the class DictionaryTypeHandlerBase, TPropertyKey is for this function only. Similar to the way it is written for other methods such as TryGetFixedProperty in this file and others in SimpleDictionaryTypeHandler.cpp

---
In reply to: [79738900](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r79738900) [](ancestors = 79738900)